backend/app/controllers/spree/admin/menus_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Spree
+  module Admin
+    class MenusController < ResourceController
+      before_action :load_data
+      before_action :load_menu_items_ordered, only: :edit
+
+      def index; end
+
+      private
+
+      def location_after_save
+        spree.edit_admin_menu_path(@menu)
+      end
+
+      private
+
+      def load_data
```

Comment:
@MatthewKennedy I think we can have `load_stores` before_action for all controllers in the admin, but maybe this should be changed to:

```ruby
Spree::Store.accessible_by(current_ability, :show)
```